### PR TITLE
docs(zh-CN): add configure connector

### PR DESCRIPTION
**Pull request by darcyYe**

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md [hunk 1]
```diff
@@ -3,3 +3,29 @@ sidebar_position: 5
...
 
 # 🪛 配置连接器
+
+_连接器_ 是 Logto 和那些提供短信邮件的服务商以及提供用户信息的社交媒体之间的桥梁。
+
+Logto 提供了很多能让用户可以访问三方服务提供商的连接器，包括了 Google，GitHub，Facebook，SendGrid 等等。
+
+当你想要使用 _连接器_ 的时候可能会感到迷茫，毕竟他是一个全新的概念。:sob:
+
+但是不要悲伤不要心急，看完这篇文档可以让你轻松拿捏！ :pinching_hand:
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:13]:
```suggestion
但是不要悲伤不要心急，看完这篇文档可以让你轻松拿捏！:pinching_hand:
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md [hunk 2]
```diff
@@ -3,3 +3,29 @@ sidebar_position: 5
...
+
+## :watch: 开始上手 :zap:
+
+让我们先从管理控制台开始吧！
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:17]:
```suggestion
让我们先从「管理控制台」开始吧！
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md [hunk 3]
```diff
@@ -3,3 +3,29 @@ sidebar_position: 5
...
+
+首先需要登录 Logto 帐号（这需要你的账号有管理员权限）。如果没有账号或者没有管理员权限的话，你需要先创建一个，详见「[创建一个管理员账户](../../tutorials/get-started/README.mdx#创建一个管理员账户)」。
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:19]:
```suggestion
首先需要登录 Logto 帐号（这需要你的账号有管理员权限）。如果没有账号或者没有管理员权限的话，你需要先创建一个，详见 [创建一个管理员账户](../../tutorials/get-started/README.mdx#创建一个管理员账户)。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md [hunk 4]
```diff
@@ -3,3 +3,29 @@ sidebar_position: 5
...
+
+点击左侧侧边栏中的「连接器」按钮 :neutral_face:
+
+:ghost: **欢迎来到 _连接器_ 的世界** :ghost:
+
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:24]: do we have a figure for this? if so would you like to add some comments

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/connector-setup-tips.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+## 短信和邮件连接器
+
+假设现在你已经在管理控制台的「连接器」页面上了，点击「配置」按钮然后选择一个 _邮件连接器_ 或 _短信连接器_ 以进入连接器的配置流程。
+
+按照顺序阅读并跟随左侧的配置指引文档，同时在右边的文本输入框里完成连接器配置的 JSON 文件。在「保存并完成」之前，你通过输入邮箱或手机号测试配置是否可以正常工作。
+
+在连接器详情页的右上角，可以进入到切换 _邮件连接器_ 或 _短信连接器_ 的页面；连接器配置的指引文档在详情页同样可以进入查看并参考。
+
+:::note
+你只能启用 **一个** _短信连接器_（或 _邮件链接器_）。
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/connector-setup-tips.md:21]:
```suggestion
你只能同时启用 **一个** _短信连接器_（或 _邮件链接器_）。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/connector-setup-tips.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+如果你更换了 _邮件连接器_ 或 _短信连接器_ 并点击了「保存并完成」，之前生效的 _邮件连接器_ 或 _短信连接器_ 就会被自动关闭掉，并且相应的配置 _config_ 就会无法回溯。<br/>
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/connector-setup-tips.md:23]:
```suggestion
如果你更换了 _邮件连接器_ 或 _短信连接器_ 并点按了「保存并完成」，之前生效的 _邮件连接器_ 或 _短信连接器_ 就会被自动关闭，并且相应的配置 _config_ 就会无法回溯。<br/>
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md [hunk 7]
```diff
@@ -3,3 +3,29 @@ sidebar_position: 5
...
+但是不要悲伤不要心急，看完这篇文档可以让你轻松拿捏！ :pinching_hand:
+
+## :watch: 开始上手 :zap:
+
+让我们先从管理控制台开始吧！
+
+首先需要登录 Logto 帐号（这需要你的账号有管理员权限）。如果没有账号或者没有管理员权限的话，你需要先创建一个，详见「[创建一个管理员账户](../../tutorials/get-started/README.mdx#创建一个管理员账户)」。
+
+点击左侧侧边栏中的「连接器」按钮 :neutral_face:
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:21]: in general 点击 -> 点按

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/connector-setup-tips.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+你可以自己妥善保存以前使用的连接器的 _config_ 以便后续还需要切换回这些连接器。
+:::
+
+当你完成了连接器的设置之后，看看如何「[启用短信或邮件验证码登录](../../tutorials/get-started/enable-passcode-sign-in)」。
+
+## 社交连接器
+
+同样从管理控制台「连接器」页面开始，去到「社交连接器」标签，添加 _社交连接器_ 开始进入配置。
```
Comment (gao-sun) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/connector-setup-tips.md:31]: > 点按右上角的「添加社交连接器」

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md (hunk 4 above)
Comment (darcyYe) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md]: We don't have the figure for this. Added some brief explanations to make the transition more smooth.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md [hunk 9]
```diff
@@ -3,3 +3,31 @@ sidebar_position: 5
 ---
 
 # 🪛 配置连接器
+
+_连接器_ 是 Logto 和那些提供短信邮件的服务商以及提供用户信息的社交媒体之间的桥梁。
```
Comment (simeng-li) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:7]: 社交媒体. social media?

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md [hunk 10]
```diff
@@ -3,3 +3,31 @@ sidebar_position: 5
...
+
+Logto 提供了很多能让用户可以访问三方服务提供商的连接器，包括了 Google，GitHub，Facebook，SendGrid 等等。
```
Comment (simeng-li) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:9]: nit: 三方身份服务提供商?

**Review by simeng-li — COMMENTED** [review]
configure connector might be complicated， we could have some screenshots here.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md (hunk 10 above)
Comment (darcyYe) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:9]: like SendGrid is not a `social identity` provider but an email service provider.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md (hunk 9 above)
Comment (darcyYe) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md]: updated to 社交媒体平台

**Review by gao-sun — APPROVED** [review]
cool

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md (hunk 9 above)
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md]: maybe just 社交平台

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md (hunk 10 above)
Comment (simeng-li) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/recipes/configure-connectors/README.md:9]: got it

**Review by wangsijie — APPROVED** [review]

**Pull request merged**